.github/workflows/main.yaml

```diff
@@ -126,6 +127,7 @@ jobs:
             type=semver,pattern={{major}}.{{minor}}
             type=semver,pattern={{major}}
             type=sha
+            type=raw,value=latest,enable={{is_default_branch}}
```

Comment:
Could we use some yaml anchors here to reduce the duplication mate?

Reply:
Could do but I don't think it's worth it right now :)